GuildedRoseLLC/ItemsViewController.swift

```diff
@@ -29,7 +35,6 @@ public class ItemsViewController: UIViewController, UICollectionViewDelegate {
     }
     
     private func configureDataSource(items: [Item]) {
-        dataSource = ItemCollectionViewDataSource()
         dataSource?.show(items: items)
```

Comment:
Is this line necessary? `configureDataSource` is being called with an empty array, which is what `ItemCollectionViewDataSource` already defaults to.

Reply:
Thanks for the feedback, implemented.